### ISSUE TITLE
Screen Icons

### ISSUE THREAD

**Issue by samreid** [phetsims/wave-interference#32]
@arouinfar can you recommend screen icons for the simulation?

**Comment by arouinfar** [comment]
@samreid I think it would be something like
* **Waves:** A propagating wave
* **Interference:** Two sources, but probably no interference pattern, as we would want students to discover that on their own
* **Slits:** Double slits, possibly with a wave approaching, but again, probably wouldn't show the interference pattern
* **Diffraction:** Laser beam shining grating, possibly with or without the laser itself.

At this stage I think things are still a bit too wireframe-y to make finalized screen icons. However, if it useful to have stand-in screen icons, I'd be happy to provide you with assets. Let me know what you'd prefer @samreid.

**Comment by samreid** [comment]
I'd recommend to add proposed/stand-in screen icons sooner rather than later, even if they are not the final ones.

**Comment by arouinfar** [comment]
@samreid I've emailed you stand-in screen icons.

**Comment by arouinfar** [comment]
After you've implemented these, please reassign this issue to me. I'll mark it as on-hold until the design is far enough along to create the final icons.

**Comment by samreid** [comment]
I added the screen icons in the preceding commit.  No need to change them now, but I thought I'd jot down my initial reactions so I don't forget:
1. Black on black is hard to parse (black background against navbar or home screen)
2. The red stands out nicely, but maybe would be more consistent to match first color in the sim, of blue (for water).
3. The slits will need to be much larger (zoomed in on) to be visible in the navbar

**Comment by arouinfar** [comment]
Thanks for the feedback @samreid!

> Black on black is hard to parse (black background against navbar or home screen)

Agreed. I've added a white stroke to the icons.

> The red stands out nicely, but maybe would be more consistent to match first color in the sim, of blue (for water).

For speed, I cloned and resused the same wave when creating the icons. Definitely agree that the Waves icon should represent the water scene.

> The slits will need to be much larger (zoomed in on) to be visible in the navbar

The barrier visibility isn't great. I'll make it a bit more cartoonish when working on the final version.

**Comment by samreid** [comment]
@arouinfar can you resume work on this now that the simulation design is further along?

**Comment by arouinfar** [comment]
@samreid I've cleaned up the icons a bit, and added a white border so they stand out against the black home screen and navbar a bit better. The updated assets are temporarily up on Drive in the Wave Interference folder. 

![screen_icons_2_waves](https://user-images.githubusercontent.com/8419308/47392209-33329380-d6d9-11e8-882e-0368e9b33930.png)

![screen_icons_2_interference](https://user-images.githubusercontent.com/8419308/47392207-33329380-d6d9-11e8-9f0f-59b78c43acd3.png)

![screen_icons_2_slits](https://user-images.githubusercontent.com/8419308/47392208-33329380-d6d9-11e8-9ab5-7af844f490f9.png)

![screen_icons_2_diffraction](https://user-images.githubusercontent.com/8419308/47392206-3299fd00-d6d9-11e8-9ece-7d8aaf6f57b7.png)

**Comment by samreid** [comment]
I updated the screen icons in the preceding commit, @arouinfar can you please review in context on phettest?  Close if all is well.  Thanks!

**Comment by arouinfar** [comment]
Looks good to me @samreid! I briefly compared Chrome and Firefox, and didn't notice any appreciable difference.

**Comment by arouinfar** [comment]
In https://github.com/phetsims/joist/issues/538, we decided to add the ability for joist to handle the stroke around navbar icons. This option already exists for home screen icons. @pixelzoom commented:

> For home screen icons, there is apparently an option to add a stroke, and it's used in SOM.  In JOIST/Sim:
```js
      // If true, there will be a border shown around the home screen icons.  Use this option if the home screen icons
      // have the same color as the background, as in Color Vision.
      showSmallHomeScreenIconFrame: false,
```
>The option name could use some work, and the doc isn't quite right.  What it really does is puts a border around unselected home screen icons.  
>
>@arouinfar You should probably use this option in Wave Interference and remove the border from the artwork.

@samreid I've removed the white outline from the screen icons, and have temporarily stored them in the sim folder. The Diffraction screen icon didn't have a stroke, so it doesn't need to be replaced.
https://drive.google.com/drive/u/0/folders/0B6CMwxdP0NGYdWdmeGo0QVhvMUk

**Comment by samreid** [comment]
Fixed in the preceding commit.  Looks great on the home screen.  Should we also do something about the navigation bar buttons, which do not have a stroke?

![image](https://user-images.githubusercontent.com/679486/48427186-75258700-e725-11e8-974d-009b44c19deb.png)

![image](https://user-images.githubusercontent.com/679486/48427230-8bcbde00-e725-11e8-94b4-11e046b92a52.png)

**Comment by arouinfar** [comment]
Thanks @samreid, looks good. The ability to add a stroke around the navbar icons will be handled by @pixelzoom in https://github.com/phetsims/joist/issues/538. Until then, we should mark this issue as on-hold.

**Comment by samreid** [comment]
Self-assigning so this will be on my list after the hold is removed.

**Comment by arouinfar** [comment]
Now that the default frequency is green (#211), we decided that the screen icons should be updated to match.

**Comment by arouinfar** [comment]
@samreid I've uploaded the updated screen icon assets to the sim design folder
https://drive.google.com/drive/u/0/folders/0B6CMwxdP0NGYdWdmeGo0QVhvMUk

**Comment by samreid** [comment]
New screen icons committed, @arouinfar can you please review?

**Comment by arouinfar** [comment]
Looks good @samreid. Marking as on-hold again until the joist work wraps up.

**Comment by samreid** [comment]
At today's design meeting, @ariel-phet and @arouinfar said this is not essential for 1.0, marking deferred.

**Comment by samreid** [comment]
@arouinfar this feature is implemented and looks like this: https://github.com/phetsims/joist/issues/538#issuecomment-454282235

Would you like me to use this option for Wave Interference?

**Comment by arouinfar** [comment]
@samreid yes, let's use this option in Wave Interference.

**Comment by samreid** [comment]
Committed, @arouinfar can you please test on phettest/master and make sure it looks and behaves as desired?

**Comment by arouinfar** [comment]
👍
